hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/GrpcOzoneManagerServer.java

```diff
@@ -91,9 +97,15 @@ public void init(OzoneManagerProtocolServerSideTranslatorPB omTranslator,
                    CertificateClient caClient) {
     NettyServerBuilder nettyServerBuilder = NettyServerBuilder.forPort(port)
         .maxInboundMessageSize(maxSize)
-        .addService(new OzoneManagerServiceGrpc(omTranslator,
-            delegationTokenMgr,
-            omServerConfig));
+        .maxConnectionIdle(CLIENT_TIMEOUT_SECS, TimeUnit.SECONDS)
```

Comment:
This timeout is questionable.  What is the reason for this.  Offline we had a discussion to not have a timeout.  Can we remove the timeout?

Reply:
@neils-dev I get that there is great advantage to keeping the client connections open, instead of opening and closing them per request as hRPC does, but IMHO there should be a long timeout. We can't keep resources occupied forever if there isn't traffic for over 30 minutes.

If we were using `ManagedChannel`, every trasport connection would be closed after 30 mins of inactivity as a default. With the `NettyServer` there is no timeout and we are holding resources forever. I think we should also have a 30 min timeout per transport. If a client is idle for such time, we shouldn't keep the connection open.